spec/features/job_seekers_can_view_vacancies_spec.rb

```diff
@@ -242,8 +242,9 @@
 
   context 'when the user is on mobile', js: true do
     scenario 'they should see the \'Show more filters\' link' do
-      page.driver.add_header('User-Agent', USER_AGENTS['MOBILE_CHROME'])
-      visit jobs_path
+      inspect_requests(inject_headers: { 'User-Agent' => USER_AGENTS['MOBILE_CHROME'] }) do
```

Comment:
I found some code on the [ChromeDriver](http://chromedriver.chromium.org/mobile-emulation) website which looks simpler:

```
mobile_emulation = {

    "deviceMetrics" => { "width" => 360, "height" => 640, "pixelRatio" => 3.0 },

    "userAgent" => "Mozilla/5.0 (Linux; Android 4.2.1; en-us; Nexus 5 Build/JOP40D) AppleWebKit/535.19 (KHTML, like Gecko) Chrome/18.0.1025.166 Mobile Safari/535.19"

}

caps = Selenium::WebDriver::Remote::Capabilities.chrome("chromeOptions" => mobile_emulation)
```

Reply:
Unfortunately, multiple different variations on that weren't working as expected.  However, when I went to copy the stackoverflow version that I'd been attempting just now (very similar to what you've pasted here), I noticed that I'd missed another version of the answer: 

https://stackoverflow.com/questions/38969593/rails-simulate-mobile-device-with-capybara

```bash 
Capybara.register_driver :iphone do |app|
  Capybara::Selenium::Driver.new(app,
    :browser => :chrome,
    :desired_capabilities => Selenium::WebDriver::Remote::Capabilities.chrome(
      'chromeOptions' => {
        'mobileEmulation' => {
          'deviceMetrics' => { 'width' => 360, 'height' => 640, 'pixelRatio' => 3.0 },
          'userAgent' => "Mozilla/5.0 (Linux; Android 4.2.1; en-us; Nexus 5 Build/JOP40D) AppleWebKit/535.19 (KHTML, like Gecko) Chrome/18.0.1025.166 Mobile Safari/535.19"
        }
      }
    )
  )
end
```
I might give that a try.  

However, the GOV.UK design toolkit is bootstrap based and fully responsive. We really shouldn't be delivering different views based on user agent to accomplish something that could be done in a much more straightforward way using breakpoints on a responsive page.